containers/src/Kokkos_DynRankView.hpp

```diff
@@ -449,10 +449,15 @@ class DynRankView : private View<DataType*******, Properties...> {
   using reference_type = typename view_type::reference_type;
   using pointer_type   = typename view_type::pointer_type;
 
-  using scalar_array_type           = value_type;
-  using const_scalar_array_type     = const_value_type;
-  using non_const_scalar_array_type = non_const_value_type;
-  using specialize                  = typename view_type::specialize;
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
+  using scalar_array_type KOKKOS_DEPRECATED_WITH_COMMENT(
+      "Use data_type instead .") = value_type;
+  using const_scalar_array_type KOKKOS_DEPRECATED_WITH_COMMENT(
+      "Use const_data_type instead .") = const_value_type;
+  using non_const_scalar_array_type KOKKOS_DEPRECATED_WITH_COMMENT(
+      "Use non_const_data_type instead .") = non_const_value_type;
```

Comment:
```suggestion
  using scalar_array_type KOKKOS_DEPRECATED_WITH_COMMENT(
      "Use value_type instead.") = value_type;
  using const_scalar_array_type KOKKOS_DEPRECATED_WITH_COMMENT(
      "Use const_value_type instead .") = const_value_type;
  using non_const_scalar_array_type KOKKOS_DEPRECATED_WITH_COMMENT(
      "Use non_const_value_type instead.") = non_const_value_type;
```
What Daniel said.  I don't care if you keep the dot but the space before must go